src/test/on_exit.cc

```diff
@@ -106,7 +106,7 @@ int main(int argc, char **argv)
     *new_val = EXIT_FUNC_VAL;
     exit_func_mgr.add_callback(exit_func_cb, new_val);
     call_exit();
-    assert(0);
+    ceph_abort();
```

Comment:
[ 64%] Building CXX object src/test/CMakeFiles/unittest_on_exit.dir/on_exit.cc.o
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/on_exit.cc: In function ‘int main(int, char**)’:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/on_exit.cc:109:16: error: ‘ceph_abort’ was not declared in this scope
     ceph_abort();